lib/tasks/pubmed.rake

```diff
@@ -10,4 +10,36 @@ namespace :pubmed do
     doc = client.fetch_records_for_pmid_list(pmids)
     puts doc # XML document
   end
+
+  desc 'Harvest using a text file with a list of publications by PubmedID for the supplied cap_profile_id'
+  task :pmid_profile_id_import, [:path_to_report,:cap_profile_id] => :environment do |_t, args|
+    lines = IO.readlines args[:path_to_report]
+    author = Author.find_by_cap_profile_id(args[:cap_profile_id])
+
+    puts "Cap_profile_id #{args[:cap_profile_id]} is #{author.first_name} #{author.last_name}"
+    total_pub_count=author.contributions.size
+    new_pub_count=author.contributions.where(:status=>'new').size
+    approved_pub_count=author.contributions.where(:status=>'approved').size
+    puts "total publications: #{total_pub_count}"
+    puts "total new publications: #{new_pub_count}"
+    puts "total accepted publications: #{approved_pub_count}"
+    puts "attempting to harvest #{lines.count} new publications by pubmedID"
+    puts
```

Comment:
It's not important here, but consider the performance tradeoffs of doing multiple SQL queries on contributions vs. getting them all back and then doing the filtered counts using a ruby `select` or `count` (in-memory), e.g.
```ruby
x = [*1..10]
# => [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
> x.count {|i| i % 2 == 0 }
#=> 5
```